src/Documentation/Documentation.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
  */
-final class Documentation
+final class Documentation implements DocumentationInterface
```

Comment:
:+1:

Reply:
I need this for the `DocumentationAction` upgrade path. In fact the new `OpenApi` class implements it so that I can still return a `src/Documentation/Documentation`.